Source/Foundation/bsfUtility/Private/UnitTests/BsUtility_bench.cpp

```diff
@@ -0,0 +1,69 @@
+//************************************ bs::framework - Copyright 2018 Marko Pintera **************************************//
```

Comment:
I like the benchmarks, would be really nice to have these going for more systems. File name should probably be `BsUtilityBench` - similar naming concerns for `_test` files.